modules/installation-aws-permissions.adoc

```diff
@@ -185,8 +189,9 @@ If you have not created a load balancer in your AWS account, the IAM user also r
 * `s3:GetBucketCors`
 * `s3:GetBucketLocation`
 * `s3:GetBucketLogging`
-* `s3:GetBucketPolicy`
 * `s3:GetBucketObjectLockConfiguration`
+* `s3:GetBucketPolicy`
+* `s3:GetReplicationConfiguration` #listed twice
```

Comment:
One of the instances of this permission can be removed. No point in listing it twitce in the same section.

Reply:
Thanks. Removed.